src/iOS.Core/Views/FormEntryTableViewCell.cs

```diff
@@ -161,6 +161,27 @@ public class FormEntryTableViewCell : ExtendedUITableViewCell, ISelectable
                 ContentView.TrailingAnchor.ConstraintEqualTo(Button.TrailingAnchor, 10f).Active = true;
                 Button.LeadingAnchor.ConstraintEqualTo(TextField.TrailingAnchor, 10f).Active = true;
             }
+            if (useTwoButtons)
+            {
+                Button = new UIButton(UIButtonType.System);
+                Button.Frame = ContentView.Bounds;
+                Button.TranslatesAutoresizingMaskIntoConstraints = false;
+                Button.SetTitleColor(ThemeHelpers.PrimaryColor, UIControlState.Normal);
+
+                SecondButton = new UIButton(UIButtonType.System);
+                SecondButton.Frame = ContentView.Bounds;
+                SecondButton.TranslatesAutoresizingMaskIntoConstraints = false;
+                SecondButton.SetTitleColor(ThemeHelpers.PrimaryColor, UIControlState.Normal);
+
+                ContentView.Add(SecondButton);
+                ContentView.Add(Button);
+
+                ContentView.BottomAnchor.ConstraintEqualTo(Button.BottomAnchor, 10f).Active = true;
+                ContentView.BottomAnchor.ConstraintEqualTo(SecondButton.BottomAnchor, 10f).Active = true;
+                SecondButton.LeadingAnchor.ConstraintEqualTo(TextField.TrailingAnchor, 9f).Active = true;
+                Button.LeadingAnchor.ConstraintEqualTo(SecondButton.TrailingAnchor, 10f).Active = true;
+                ContentView.TrailingAnchor.ConstraintEqualTo(Button.TrailingAnchor, 10f).Active = true;
+            }
```

Comment:
Refactor this and the part above of using one button to use the enum and extract the logic away from this method:
```c#
if (buttonsConfig != ButtonsConfig.None)
{
     AddButtons(buttonsConfig);
}
```
Also reuse code when possible, like here that you're duplicating code to configure the first button on the one button and two buttons scenarios, as well as some constraints that are the same.

Oh and btw, you can use `NSLayoutConstraint.ActivateConstraints(new NSLayoutConstraint[] { });` to pass all the constraints and it will activate all of them instead of having to set `.Active = true` on each of them